config/language/es/views.view.frontpage.yml

```diff
@@ -21,7 +22,7 @@ display:
             items_per_page_options_all_label: '— Todo(s) —'
             offset_label: Desplazamiento
           tags:
-            previous: '‹ Previous'
+            next: 'Siguiente ›'
```

Comment:
Why Spanish, @nuez? Let's keep everything in English and then we can translate. Deal?